docs/fundamentals/integrations-overview.md

```diff
@@ -14,19 +14,119 @@ ms.topic: conceptual
 
-While .NET Aspire _client integrations_ can be used independently, they work best with the [.NET Aspire app host](app-host-overview.md). Each [integration](#available-integrations) provides installation, configuration, and usage details in the documentation—for both sides of the integration, the hosting and client respectively.
+Hosting integrations extend the <xref:Aspire.Hosting.IDistributedApplicationBuilder> interface, enabling the _app host_ project to express resources within its _app model_. The official [hosting integration NuGet packages](https://www.nuget.org/packages?q=owner%3A+aspire+tags%3A+aspire+hosting+integration&includeComputedFrameworks=true&prerel=true&sortby=relevance) are tagged with `aspire`, `integration`, and `hosting`.
+
+For information on creating a custom hosting integration, see [Create custom .NET Aspire hosting integration](../extensibility/custom-hosting-integration.md).
+
+### Client integrations
+
+Client integrations define configuration schema, wire up client libraries to dependency injection (DI), add health checks, resiliency, and telemetry where applicable. These packages configure existing libraries to connect to hosting integrations. They extend the <xref:Microsoft.Extensions.DependencyInjection.IServiceCollection> interface allowing client-consuming projects to use the connected resource. The official [client integration NuGet packages](https://www.nuget.org/packages?q=owner%3A+aspire+tags%3A+aspire+client+integration&includeComputedFrameworks=true&prerel=true&sortby=relevance) are tagged with `aspire`, `integration`, and `client`.
+
+For more information on creating a custom client integration, see [Create custom .NET Aspire client integration](../extensibility/custom-client-integration.md).
+
+### Relationship between hosting and client integrations
+
+Hosting and client integrations are best when used together, but are **not** coupled and may be used separately. Some hosting integrations may not have a corresponding client integration. Configuration is what makes the hosting integration work with the client integration.
 
 For example, if you want to use [Redis for caching](../caching/stackexchange-redis-caching-overview.md) in your .NET Aspire solution, you would use the `Aspire.Hosting.Redis` package to model the Redis resource. Then, you can connect to the Redis resource using the `Aspire.StackExchange.Redis` client integration.
 
-## Available integrations
+## Available hosting integrations
+
+The following section details the available .NET Aspire hosting integrations, links to their respective NuGet packages, the resource types they expose, and provides a brief description of each integration.
+
+**Cloud-agnostic resources are available in the following NuGet packages:**
```

Comment:
Can we unify the layout for client and hosting integrations, either have both look like this or have both look like a table.

Reply:
I prefer having these look like how client integrations look.

Reply:
Ah, yes... good callout. What do you think about having a layout that includes a table with both sides? Showing which hosting integrations aligns with each client integration?

Reply:
Yes, that works. The ones without are just in the table then?

Reply:
Yeah, the idea is to quickly visualize which integrations are end-to-end, and which are individually either end. This will help to understand gaps/opportunities.